### PR TITLE
Add printing failed message when timeout

### PR DESCRIPTION
**Pull request by sophalHong**
Signed-off-by: sophal <sophalHong@github.com>

# Describe what this PR does #

Base on the previous particular script implementation, when something does not work well, it will run again and again in for-loop until timeout is over. Once timeout is over, it will break the loop, and continue to the step without notifying that it is failed.
so this PR just added few lines of script code to check whether timeout is over and then printing failed message.

## Is there anything that requires special attention ##

Do you have any questions?
No

Is the change backward compatible?
No

Are there concerns around backward compatibility?
No

Provide any external context for the change, if any.
No

## Future concerns ##

List items that are not part of the PR and do not impact it's
functionality, but are work items that can be taken up subsequently.

### CODE REVIEW

File: scripts/rook.sh [hunk 1]
```diff
@@ -51,6 +51,11 @@ function check_ceph_cluster_health() {
 			fi
 		fi
 	done
+
+	if [ "$retry" -gt "$ROOK_DEPLOY_TIMEOUT" ]; then
+		echo "[ERROR] Check CEPH Cluster state failed! (timeout)"
+		exit
```
Comment (Madhu-1) [inline scripts/rook.sh]: echo "[Timeout] CEPH cluster not  in a healthy state (timeout)"
exit 1

File: scripts/rook.sh [hunk 2]
```diff
@@ -72,6 +77,10 @@ function check_mds_stat() {
 			fi
 		fi
 	done
+
+	if [ "$retry" -gt "$ROOK_DEPLOY_TIMEOUT" ]; then
+		echo "[ERROR] Check CEPH Filesystem state failed! (timeout)"
```
Comment (Madhu-1) [inline scripts/rook.sh]: echo "[Timeout]  Failed to get ceph filesystem pods"
exit 1

File: scripts/rook.sh [hunk 3]
```diff
@@ -82,14 +91,18 @@ function check_rbd_stat() {
...
+		[[ "$TOOLBOX_POD_STATUS" != "Running" ]] && \
+			{ echo "Toolbox POD ($TOOLBOX_POD) status: [$TOOLBOX_POD_STATUS]"; continue; }
 
 		if kubectl exec -n rook-ceph "$TOOLBOX_POD" -it -- rbd pool stats "$RBD_POOL_NAME" &>/dev/null; then
 			echo "RBD ($RBD_POOL_NAME) is successfully created..."
 			break
 		fi
 	done
+
+	if [ "$retry" -gt "$ROOK_DEPLOY_TIMEOUT" ]; then
```
Comment (Madhu-1) [inline scripts/rook.sh]: echo "[Timeout] Failed to get RBD pool  stats"
exit 1

**Comment by humblec** [comment]
LGTM. Thanks !

**Pull request merged**